modules/repository/generate.go

```diff
@@ -182,6 +250,15 @@ func generateRepoCommit(repo, templateRepo, generateRepo *models.Repository, tmp
 		return fmt.Errorf("git remote add: %v", err)
 	}
 
+	// Reapply the submodules by updating the index
```

Comment:
we may wana move this into the `modules/git` package ... and we should also utilize if exist open pipes to git comans

as @zeripath converted a lot of them to improve speed ...

Reply:
By utilizing pipes do you mean using `RunInDirPipeline` instead of `RunInDir` for the command or is there more to it?